client/me/purchases/purchase-item/index.jsx

```diff
@@ -192,7 +192,7 @@ class PurchaseItem extends Component {
 		const { purchase, translate } = this.props;
 
 		if ( purchase && isPartnerPurchase( purchase ) ) {
-			return translate( 'This plan is managed by %(partnerName)s', {
+			return translate( 'This plan is managed by %(partnerName)s.', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.